docs/5.0/preview/database-access.md

```diff
@@ -44,23 +44,82 @@ The following diagram shows an example Database Access setup:
 
 ## Schedule
 
-!!! info
-    Teleport Database Access is under active development.
-    The alpha release will include support for PostgreSQL, including Amazon RDS and Aurora.
+Teleport Database Access is under active development. The alpha release will
+include support for PostgreSQL, including Amazon RDS and Aurora.
 
-[Database Access Release Schedule](./upcoming-releases.md#release-schedule)
+See [release schedule](./upcoming-releases.md#release-schedule).
 
-## Setup
+## Configure PostgreSQL
 
-Let's setup a sample Teleport Database Access deployment.
+### On-Prem PostgreSQL
```

Comment:
```suggestion
### Self-Hosted PostgreSQL
```

We are using self-hosted vs on-prem. As on-prem can be a limited term vs 'self-hosted' which includes people running PostgreSQL themselves on a cloud provider.